### PR TITLE
Add org.gnuhealth.mygnuhealth

### PR DESCRIPTION
**Pull request by alcir**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. Link: https://lists.gnu.org/archive/html/health/2022-01/msg00000.html
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by alcir** [comment]
bot, build org.kde.mygnuhealth

**Comment by nedrichards** [comment]
More appdata needed! (which will also make the app work well with https://apps.kde.org/ and other systems)

> • tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
• style-invalid         : Not enough <screenshot> tags, minimum is 1
• tag-missing           : <release> required
• tag-missing           : <url> is not present

**Comment by alcir** [comment]
bot, build org.kde.mygnuhealth

**Comment by g-fb** [comment]
mygnuhealth is not a kde project anymore.

**Comment by alcir** [comment]
> mygnuhealth is not a kde project anymore.

Out of curiosity, do you have any reference?

And, so I should rename it? Because in the current version tar.gz file, there is org.kde.mygnuhealth (desktop file as well metainfo, etc.) while in the sources repository there is now org.gnuhealth.my

**Comment by g-fb** [comment]
> Out of curiosity, do you have any reference?

I asked about it in https://webchat.kde.org/#/room/#chat:kde.org, since the repo wasn't available in invent.kde.org.

You can search the [room](https://webchat.kde.org/#/room/#chat:kde.org) for mygnuhealth.

> And, so I should rename it? Because in the current version tar.gz file, there is org.kde.mygnuhealth (desktop file as well metainfo, etc.) while in the sources repository there is now org.gnuhealth.my

That I don't know.

**Comment by alcir** [comment]
> while in the sources repository there is now org.gnuhealth.my

Whoops, I looked in the wrong place.

http://hg.savannah.gnu.org/hgweb/health-mygnuhealth/file
Looks like they are still `org.kde.mygnuhealth`

**Comment by alcir** [comment]
bot, build org.kde.mygnuhealth

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: org.kde.mygnuhealth.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+base-version: '5.15-21.08'
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm12
+command: mygnuhealth
+rename-icon: mygnuhealth
+finish-args:
+  - --socket=x11
```
[the remark this answers is not in the document]
Comment (alcir) [reply org.kde.mygnuhealth.yaml]: Does the latest commit look better?

**Comment by nedrichards** [comment]
You should also send the appdata patch upstream as this is useful for all linux users, not just flatpak.

**Comment by alcir** [comment]
> You should also send the appdata patch upstream as this is useful for all linux users, not just flatpak.

I agree.
Look this thread
https://lists.gnu.org/archive/html/health/2022-01/msg00008.html

**Comment by alcir** [comment]
bot, build org.kde.mygnuhealth

**Comment by alcir** [comment]
bot, build org.gnuhealth.mygnuhealth

**Comment by nedrichards** [comment]
> > You should also send the appdata patch upstream as this is useful for all linux users, not just flatpak.
> 
> I agree. Look this thread https://lists.gnu.org/archive/html/health/2022-01/msg00008.html

OK, hrm, looking at this the whole app should be org.gnuhealth.mygnuhealth? It's pretty annoying for users to make that change after we publish it so I'll hold until this update lands.

**Comment by barthalion** [comment]
FWIW, you can rename it now, as long as it doesn't actually use its D-Bus name for anything.

**Comment by nedrichards** [comment]
From my initial look I don't think it does - GNU Savannah is not the most... quick glance friendly platform for a drive by review though.

**Comment by alcir** [comment]
bot, build org.gnuhealth.mygnuhealth

File: org.gnuhealth.mygnuhealth.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  - python3-bcrypt.yaml
+  - python3-tinydb.yaml
+  - python3-requests.yaml
+  - name: mygnuhealth
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix=/app --no-deps .
+    sources:
+      - type: archive
+        url: http://hg.savannah.gnu.org/hgweb/health-mygnuhealth/archive/tip.tar.gz
```
Comment (hfiguiere) [inline org.gnuhealth.mygnuhealth.yaml:30]: This is an ever changing content. You should pin a specific revision / tag.

Comment (alcir) [reply org.gnuhealth.mygnuhealth.yaml:30]: Yes. It was only a test.
Probably upstream will release version 1.0.6 soon, following the proposed changes to appdata and desktop file.

**Pull request closed without merging**